mods/ssb/moves.js

```diff
@@ -3464,8 +3468,8 @@ let BattleMovedex = {
 	rotate: {
 		accuracy: 100,
 		category: "Status",
-		desc: "The next Pokemon will switch out after using its move if the move is successful.",
-		shortDesc: "Next Pokemon will switch after using its move.",
+		desc: "The user's replacement will switch out after using their move on the next turn if the replacement’s move is successful",
```

Comment:
```suggestion
		desc: "The user's replacement will switch out after using their move on the next turn if the replacement's move is successful.",
```
don't use curly apostrophes also there should be a period at the end of the sentence